docs/document/content/user-manual/shardingsphere-scaling/usage.cn.md

```diff
@@ -184,13 +204,25 @@ mysql> show scaling status 660152090995195904;
 | PREPARING                                         | 准备中                                                        |
 | RUNNING                                           | 运行中                                                        |
 | EXECUTE_INVENTORY_TASK                            | 全量迁移中                                                     |
-| EXECUTE_INCREMENTAL_TASK                          | 增量迁移中                                                     |
-| ALMOST_FINISHED                                   | 基本完成                                                       |
-| FINISHED                                          | 已完成                                                         |
+| EXECUTE_INCREMENTAL_TASK                          | 增量迁移中                                                     | |
```

Comment:
Seems the end of line includes a duplicated "|".

P.S. You could preview the rendered page in markdown editor, make sure every element works.

Reply:
sorry, I will check next time.